system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -0,0 +1,222 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <cmath>
+#include <memory>
+#include <fstream>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+
+#include "metrics_statistics_msgs/msg/metrics_message.hpp"
+#include "metrics_statistics_msgs/msg/statistic_data_type.hpp"
+
+#include "../../src/system_metrics_collector/linux_process_cpu_measurement_node.hpp"
```

Comment:
not use relative include paths?

Reply:
This case would require additional `-I` settings.

We should open an issue tracking the to-be separate PR for fixing these relative include paths.

Let's leave fixing this case to the said separate PR.

Reply:
Said issue is opened here https://github.com/ros-tooling/system_metrics_collector/issues/60.